Resources/doc/reference/glossary.rst

```diff
@@ -5,9 +5,7 @@ Glossary
 
     Cacheable
         A *response* is considered cacheable when the status code is one of
-        200, 203, 300, 301, 302, 404, 410. This range of status codes can be
-        extended with :ref:`additional_cacheable_status` or overridden with
-        :ref:`match_response`.
+        200, 203, 300, 301, 302, 404, 410.
```

Comment:
erm, this does not match the list in other places. afaik its the same list that includes 414 and 501, no?